app/questionnaire/questionnaire_manager.py

```diff
@@ -43,7 +42,7 @@ def validate_all_answers(self):
             is_valid = self.validate(location, answers)
 
             if not is_valid:
-                logger.debug("Failed validation with current location %s", str(location))
+                logger.debug("failed validation", **location.__dict__)
```

Comment:
The only thing with that is we'd have a location in the message which looks like 
```
{
  "event": "failed validation"
  "location": "group/block/answer"
}
```
vs
```
{
"event": "failed validation"
"group": "group_id',
"block": "block_id",
"answer": "answer_id"
}
```

The latter might make it easier to query in the logs? i.e. find all log messages for group X